### PR TITLE
Show comments in the admin/instances page

### PR DESCRIPTION
**Pull request by tribela**
We have lots of spaces on admin/instances page. Show internal, public comment will be useful when managing instance blocks
image

### CODE REVIEW

File: app/views/admin/instances/_instance.html.haml [hunk 1]
```diff
@@ -7,6 +7,8 @@
       %small
         - if instance.domain_block
           = instance.domain_block.policies.map { |policy| t(policy, scope: 'admin.instances.content_policies.policies') }.join(' · ')
+          %span.public-comment= t(instance.domain_block.public_comment) if instance.domain_block.public_comment.present?
+          %span.private-comment= t(instance.domain_block.private_comment) if instance.domain_block.private_comment.present?
```
Comment (ClearlyClaire) [inline app/views/admin/instances/_instance.html.haml:11]: Those aren't translatable strings, so we shouldn't call `t()` on them:
```suggestion
          %span.public-comment= instance.domain_block.public_comment if instance.domain_block.public_comment.present?
          %span.private-comment= instance.domain_block.private_comment if instance.domain_block.private_comment.present?
```

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]
Hi! Thank you for your contribution! Unfortunately, I think it looks unclear that those are the public and private comments you set. Additionally, I think they lack separators. Finally, I'm afraid long descriptions are going to cause issues with limited width layouts:
image

**Comment by jazmichaelking** [comment]
Would something like this be more useful:

image

**Comment by ClearlyClaire** [comment]
Yeah, I think so! Although I'm slightly worried about space, but that's a minor issue. I'd also probably omit the line rather than write “None Found”.

**Comment by tribela** [comment]
image

File: app/views/admin/instances/_instance.html.haml [hunk 2]
```diff
@@ -7,6 +7,10 @@
       %small
         - if instance.domain_block
           = instance.domain_block.policies.map { |policy| t(policy, scope: 'admin.instances.content_policies.policies') }.join(' · ')
+          - if instance.domain_block.public_comment.present?
+            %span.comment.public-comment "#{t('admin.domain_blocks.public_comment')}: #{instance.domain_block.public_comment}"
+          - if instance.domain_block.private_comment.present?
+            %span.comment.private-comment "#{t('admin.domain_blocks.private_comment')}: #{instance.domain_block.private_comment}"
```
Comment (ClearlyClaire) [inline app/views/admin/instances/_instance.html.haml:13]: There are extra quotes around the lines, this should be fixed with the following:
```suggestion
          - if instance.domain_block.public_comment.present?
            %span.comment.public-comment #{t('admin.domain_blocks.public_comment')}: #{instance.domain_block.public_comment}
          - if instance.domain_block.private_comment.present?
            %span.comment.private-comment #{t('admin.domain_blocks.private_comment')}: #{instance.domain_block.private_comment}
```

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

File: app/views/admin/instances/_instance.html.haml (hunk 2 above)
Comment (tribela) [reply app/views/admin/instances/_instance.html.haml]: Thank you!

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**